backend/cloud_inquisitor/utils.py

```diff
@@ -133,10 +134,7 @@ def get_template(template):
     Returns:
         A Jinja2 Template object
     """
-    # Local import to prevent app startup failures
-    from cloud_inquisitor import app
-
-    tmplpath = os.path.join(app.config.get('BASE_PATH'), 'templates')
+    tmplpath = os.path.join(pkg_resources.resource_filename('cloud_inquisitor', 'data'), 'templates')
```

Comment:
while outside this commit may want to modify the e-mail recipient in the block above